### PR TITLE
Fix typo in README.md

### PR DESCRIPTION
**Pull request by legoktm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -506,7 +506,7 @@ The following list is a complete list of features that can be opt in and out.
 
 | Name    |  Description | Additional Dependency |Default?|
 |---------|--------------|--------|------------|
-| datetime | Eanble the date and time coordinate support | chrono | Yes |
```
Comment (38) [inline README.md:509]: This readme is automatically generated with doc-template/readme.template.md. If you don't mind could you please change it there and run the doc-template/update-readme.sh please. Thanks!

**Review by 38 — CHANGES_REQUESTED** [review]

**Comment by LollipopFt** [comment]
there's a typo in readme:

- How to check if a backend writes file successfully ?

The behavior of Plotters backend is consistent with stadnard library...

**Comment by jayvdb** [comment]
Hey @legoktm , this PR is really old.  Probably best to just drop it.

**Comment by jayvdb** [comment]
And likely already fixed by https://github.com/plotters-rs/plotters/pull/570

**Comment by legoktm** [comment]
> And likely already fixed by #570

Hey, good to see you around :) Awesome, I had totally forgotten about this.

**Pull request closed without merging**